app/plugins/content/qubesmacros/migrations/Migration20190429000000PlgContentQubesmacros.php

```diff
@@ -8,7 +8,7 @@
 /**
  * Migration script for adding Content - QUBES Macros
  **/
-class Migration20180530000000PlgContentQubesmacros extends Base
+class Migration20190429000000PlgContentQubesmacros extends Base
```

Comment:
Heads up, Bharath:  we don't need to change the date of the migration script for every new item or edit to the extension.